node/silkworm/db/access_layer.cpp

```diff
@@ -396,7 +395,7 @@ std::optional<uint64_t> read_previous_incarnation(mdbx::txn& txn, const evmc::ad
     }
 
     auto src{db::open_cursor(txn, table::kIncarnationMap)};
-    if (auto data{src.find(mdbx::slice{address.bytes, sizeof(evmc::address)}, /*throw_notfound*/ false)}; data.done) {
+    if (auto data{src.find(to_slice(address), /*throw_notfound=*/false)}; data.done) {
         assert(data.value.length() == 8);
```

Comment:
Maybe replace this with SILKWORM_ASSERT ?

Reply:
Sure; done in https://github.com/torquem-ch/silkworm/pull/494/commits/ef07eea0b72edaf64b11ac1336b1af8b30b24e9a